### PR TITLE
update UDF docs to match UI

### PR DESCRIPTION
**Pull request by kathweinschenkprophecy**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208694855966542

### CODE REVIEW

File: docs/Spark/extensibility/user-defined-functions.md [hunk 1]
```diff
@@ -9,46 +9,32 @@ tags:
...
-  },
-  {
-    "image":"/img/udf/3.png",
-    "description":<h3 style={{padding:'10px'}}>Step 3 - UDFs can now be called by their defined names</h3>,
-  },
-];
-
-<App ImageData={ImageData}></App>
-```
+1. Create a new function. You'll find the **Functions** section in the left sidebar of a project page.
```
Comment (alexanderahn) [inline docs/Spark/extensibility/user-defined-functions.md:27]: nit: "You'll" > "You can"

File: docs/Spark/extensibility/user-defined-functions.md [hunk 2]
```diff
@@ -9,46 +9,32 @@ tags:
...
-| Definition              | Definition of the UDF function. <br/> Eg: `udf((value:Int)=>value*value)`                                                                | True     |
-| UDF initialization code | Code block that contains initialization of entities used by UDFs. This could for example contain any static mapping that a UDF might use | False    |
+| Parameter               | Description                                                                                                                                 | Required |
+| :---------------------- | :------------------------------------------------------------------------------------------------------------------------------------------ | :------- |
+| Function name           | The name of the function as it appears in your project.                                                                                     | True     |
+| UDF Name                | Name of the UDF that will register it. All calls to the UDF will use this name.                                                             | True     |
```
Comment (alexanderahn) [inline docs/Spark/extensibility/user-defined-functions.md:19]: Should we also use "The name" for the UDF Name Description?

File: docs/Spark/extensibility/user-defined-functions.md [hunk 3]
```diff
@@ -9,46 +9,32 @@ tags:
...
+| Definition              | Definition of the UDF function. <br/> Eg: `udf((value:Int)=>value*value)`                                                                   | True     |
```
Comment (alexanderahn) [inline docs/Spark/extensibility/user-defined-functions.md:20]: I know this is outside the scope of your changes, but let's update "Eg" to "E.g.". Or better yet, use something like "For example".

**Review by alexanderahn — APPROVED** [review]
Left minor notes. Otherwise, LGTM

File: docs/Spark/extensibility/user-defined-functions.md (hunk 3 above)
Comment (kathweinschenkprophecy) [reply docs/Spark/extensibility/user-defined-functions.md:20]: Definitely. I'm pro-`For Example` 100000%

File: docs/Spark/extensibility/user-defined-functions.md (hunk 2 above)
Comment (kathweinschenkprophecy) [reply docs/Spark/extensibility/user-defined-functions.md:19]: Yes true

File: docs/Spark/extensibility/user-defined-functions.md (hunk 1 above)
Comment (kathweinschenkprophecy) [reply docs/Spark/extensibility/user-defined-functions.md:27]: will change

**Pull request merged**